### PR TITLE
Sept meetup update, along with updates on README, some smaller hygiene stuff

### PR DESCRIPTION
**Pull request by amarjeet000**
Major changes:
1. New post about Sept hack session
2. Github repo's README

Minor changes:
- Added author in the 1st post
- Contact info in 'About' section
- Some minor changes in contribute section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,59 @@
 # Bangalore Clojure Group Website
 
-To be updated!
+[Visit the Website](https://bangalore-clj.github.io/)
+
+
+## About
+
+The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
```
Comment (kumarshantanu) [inline README.md]: The meetups started much earlier than 2013 - check https://groups.google.com/forum/#!forum/bangalore-clj older posts for details.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+
+Our meetups usually have talks by clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists & functional freaks without any bais.
```
Comment (kumarshantanu) [inline README.md]: s/bais/bias

File: README.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+
+This website has been built using [Cryogen](http://cryogenweb.org/index.html). You can visit the detailed documentation at [Cryogen documentation](http://cryogenweb.org/docs/home.html) page in case you have any doubts. Or, you can email us on `bangaloreclojure@gmail.com`
+
+
+## Writing a post (article)
+
+To add a post, please put your content inside a markdown (.md) file. Name that file as `<YYYY-MM-DD>-file-name.md`
+Then, put this file in the `resources/templates/md/posts` directory.
+
+You can add a seetings map on top of the markdown (.md) file as below:
```
Comment (kumarshantanu) [inline README.md]: s/seetings/settings

File: README.md [hunk 4]
```diff
@@ -1,3 +1,59 @@
...
+ :author "your-name"}
+```
+
+
+## Creating a page (ex: About, Contribute, etc.)
+
+To add a page, please put your content inside a markdown (.md) file. Name that file as `file-name.md`
+Then, put this file in the `resources/templates/md/pages` directory.
+
+You can add a seetings map on top of the MD file as below:
```
Comment (kumarshantanu) [inline README.md]: s/seetings/settings

File: resources/public/pages-output/about/index.html [hunk 5]
```diff
@@ -67,7 +67,7 @@
...
+    <p>The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.</p><p>Our meetups usually have talks by clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)</p><p>Please reach out to us if you have any thoughts on what we can do together or how we can help one another.</p><p>Ways to reach us:</p><ul><li><a href='https://clojurians.herokuapp.com/'>Clojurians slack</a>: <code>#bangalore-clj</code> channel</li><li><a href='https://groups.google.com/forum/#!forum/bangalore-clj'>Google group</a></li><li>Email: <code>bangaloreclojure@gmail.com</code></li></ul><p>Happy Clojuring !</p><p>Some of our meetup venue sponsors are:</p><ul><li>SAP Concur</li><li>Quintype</li><li>Nilenso</li><li>Go-Jek</li></ul>
```
Comment (kumarshantanu) [inline resources/public/pages-output/about/index.html]: The meetups started much earlier than 2013 - check https://groups.google.com/forum/#!forum/bangalore-clj older posts for details.

File: resources/templates/md/pages/about.md [hunk 6]
```diff
@@ -3,13 +3,20 @@
  :page-index 1
  :navbar? true}
 
-The Bangalore Clojure Group hosts one of the most active (since 2013) meetups for Clojure in India, each of our meetups are conducted in different locations with different set of speakers every month.Our meetups are scheduled conveniently on every first Saturday of the month.
+The Bangalore Clojure Group hosts one of the most active (since 2013) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
```
Comment (kumarshantanu) [inline resources/templates/md/pages/about.md]: Meetups started much earlier than 2013 - check https://groups.google.com/forum/#!forum/bangalore-clj older posts for details.

File: resources/templates/md/pages/about.md [hunk 7]
```diff
@@ -3,13 +3,20 @@
...
 
-Our meetups usually have talks by clojurists, but we also do organise hackathons on a regular basis. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)
+Our meetups usually have talks by clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)
```
Comment (kumarshantanu) [inline resources/templates/md/pages/about.md]: Consider making `c` in `clojursists` upper case for standardisation and consistency. Also, consider replacing `&` with `and`. And, s/bais/bias. Make `w` in `windows` upper case and end the sentence with a period.

**Review by kumarshantanu — CHANGES_REQUESTED** [review]
Why are the `target` files checked in? Beside that point I have noted few issues for your consideration.

File: README.md (hunk 2 above)
Comment (amarjeet000) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (amarjeet000) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (amarjeet000) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (amarjeet000) [reply README.md]: fixed

File: resources/public/pages-output/about/index.html (hunk 5 above)
Comment (amarjeet000) [reply resources/public/pages-output/about/index.html]: fixed

File: resources/templates/md/pages/about.md (hunk 6 above)
Comment (amarjeet000) [reply resources/templates/md/pages/about.md]: fixed

File: resources/templates/md/pages/about.md (hunk 7 above)
Comment (amarjeet000) [reply resources/templates/md/pages/about.md]: fixed

File: README.md [hunk 8]
```diff
@@ -1,3 +1,59 @@
 # Bangalore Clojure Group Website
 
-To be updated!
+[Visit the Website](https://bangalore-clj.github.io/)
+
+
+## About
+
+The Bangalore Clojure Group hosts one of the most active (since 2010) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
```
Comment (kumarshantanu) [inline README.md]: s/Each/All

File: README.md [hunk 9]
```diff
@@ -1,3 +1,59 @@
...
+
+Our meetups usually have talks by Clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists and functional freaks without any bias.
```
Comment (kumarshantanu) [inline README.md]: Consider s/freaks/enthusiasts

File: README.md [hunk 10]
```diff
@@ -1,3 +1,59 @@
...
+
+Please reach out to us if you have any thoughts on what we can do together or how we can help one another.
+
+Ways to reach us:
+- [clojurians slack](https://clojurians.herokuapp.com/): #bangalore-clj channel
+- [Google group](https://groups.google.com/forum/#!forum/bangalore-clj)
+- Email: `bangaloreclojure@gmail.com`
+
+
+## CONTRIBUBTE
```
Comment (kumarshantanu) [inline README.md]: Make it standard case instead of upper case for consistency?

File: README.md [hunk 11]
```diff
@@ -1,3 +1,59 @@
...
+
+Please send a pull-request for any contribution. Below are different ways to contribute. Before sending PRs, please test the changes well: `lein ring server`
```
Comment (kumarshantanu) [inline README.md]: Consider adding a note on how to test the changes. Are there unit tests, or just manual testing?

File: README.md [hunk 12]
```diff
@@ -1,3 +1,59 @@
...
+You can add a settings map on top of the MD file as below:
+```
+{:title "Contribute"
+ :layout :page
+ :page-index 0
+ :navbar? true}
+```
+
+
+## Making changes to the website (structure, looks, etc)
```
Comment (kumarshantanu) [inline README.md]: There should be no comma before `etc.`

File: resources/public/pages-output/about/index.html [hunk 13]
```diff
@@ -67,7 +67,7 @@
...
+    <p>The Bangalore Clojure Group hosts one of the most active (since 2010) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.</p><p>Our meetups usually have talks by Clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists and functional freaks without any bias (We also welcome Windows users 🙂)</p><p>Please reach out to us if you have any thoughts on what we can do together or how we can help one another.</p><p>Ways to reach us:</p><ul><li><a href='https://clojurians.herokuapp.com/'>Clojurians slack</a>: <code>#bangalore-clj</code> channel</li><li><a href='https://groups.google.com/forum/#!forum/bangalore-clj'>Google group</a></li><li>Email: <code>bangaloreclojure@gmail.com</code></li></ul><p>Happy Clojuring !</p><p>Some of our meetup venue sponsors are:</p><ul><li>SAP Concur</li><li>Quintype</li><li>Nilenso</li><li>Go-Jek</li></ul>
```
Comment (kumarshantanu) [inline resources/public/pages-output/about/index.html]: s/Each/All

File: resources/templates/md/pages/about.md [hunk 14]
```diff
@@ -3,13 +3,20 @@
  :page-index 1
  :navbar? true}
 
-The Bangalore Clojure Group hosts one of the most active (since 2013) meetups for Clojure in India, each of our meetups are conducted in different locations with different set of speakers every month.Our meetups are scheduled conveniently on every first Saturday of the month.
+The Bangalore Clojure Group hosts one of the most active (since 2010) Clojure meetups in India. Each of our meetups are conducted at different locations with different set of speakers every month. Our meetups are scheduled conveniently on every first Saturday of the month.
 
-Our meetups usually have talks by clojurists, but we also do organise hackathons on a regular basis. We are an open community, we encourage all Clojurists & functional freaks without any bais (We also welcome windows users 🙂)
+Our meetups usually have talks by Clojurists, but we organise hackathons on regular basis as well. We are an open community, we encourage all Clojurists and functional freaks without any bias (We also welcome Windows users 🙂)
```
Comment (kumarshantanu) [inline resources/templates/md/pages/about.md]: Consider s/freaks/enthusiasts and put a full stop after "Windows users".

File: resources/templates/md/posts/2018-09-10-sept-meetup.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
+{:title "The hack-session meetup on Sept 1, 2018"
+ :layout :post
+ :tags  ["Clojure" "ClojureScript"]
+ :author "Amarjeet"}
+
+
+We hosted the 2nd episode of the hack-session during this [Sept meetup](https://www.meetup.com/Bangalore-Clojure-User-Group/events/253965766/) at Nilenso office. 1st episode was during the [Aug meetup](https://www.meetup.com/Bangalore-Clojure-User-Group/events/252993263/).
+
+Not only we had larger participation, Clojurians could also push their PRs to open-sourced projects. And, some are in production already 🙂
```
Comment (kumarshantanu) [inline resources/templates/md/posts/2018-09-10-sept-meetup.md]: Consider dropping "And," as the first word of the last sentence. Also, put a full stop after the last sentence.

**Review by kumarshantanu — CHANGES_REQUESTED** [review]
Added few more proof-reading observations.

File: README.md (hunk 8 above)
Comment (amarjeet000) [reply README.md]: Fixed

File: README.md (hunk 9 above)
Comment (amarjeet000) [reply README.md]: Fixed

File: README.md (hunk 10 above)
Comment (amarjeet000) [reply README.md]: Fixed

File: README.md (hunk 11 above)
Comment (amarjeet000) [reply README.md]: Haven't really explored the unit tests options with Cryogen. For now, just following the manual testing process. I should dig into a bit. Will update this section in future accordingly.

File: README.md (hunk 12 above)
Comment (amarjeet000) [reply README.md]: I think, gramatically there should be a comma :) . But, a quick search showed that there are some debbates on this in terms of when to put comma before `etc`. But, when in doubt, use comma.

File: resources/public/pages-output/about/index.html (hunk 13 above)
Comment (amarjeet000) [reply resources/public/pages-output/about/index.html]: Fixed

File: resources/templates/md/pages/about.md (hunk 14 above)
Comment (amarjeet000) [reply resources/templates/md/pages/about.md]: Fixed. Removed "Windows users" thing. I wasn't sure if this comment will triiger a different emotion in someone's mind.

File: resources/templates/md/posts/2018-09-10-sept-meetup.md (hunk 15 above)
Comment (amarjeet000) [reply resources/templates/md/posts/2018-09-10-sept-meetup.md]: Fixed.

**Comment by Robert-Christopher** [comment]
@kumarshantanu I guess @amarjeet000 has resolved the suggestions from your review, could just quickly run through the changes ?

**Review by kumarshantanu — APPROVED** [review]

**Pull request merged**